.github/workflows/release-linux-glibc-2-28.yml

```
@@ -0,0 +1,79 @@
+          # Sanitize version for use in filenames (replace path separators)
+          version=${version//\//-}
+          base=$(pwd)/slang-${version}-linux-${{ matrix.platform }}-glibc-2.28
+          sudo mv "$(pwd)/build/dist-release/slang.zip" "${base}.zip"
+          echo "SLANG_BINARY_ARCHIVE_ZIP=${base}.zip" >> "$GITHUB_OUTPUT"
+          sudo mv "$(pwd)/build/dist-release/slang.tar.gz" "${base}.tar.gz"
+          echo "SLANG_BINARY_ARCHIVE_TAR=${base}.tar.gz" >> "$GITHUB_OUTPUT"
+      - name: File check
+        run: |
+          find "build/dist-release" -type f ! -iname '*.md' ! -iname '*.h' -print0 | xargs -0 file
+      - name: UploadBinary
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)